Makefile

```diff
@@ -1,6 +1,7 @@
 arch ?= x86_64
 target ?= $(arch)-unknown-hermit
 release ?= 0
+app ?= "$(PWD)/data/hello_world"
```

Comment:
Is this just for testing purposes? `app` should depend on `arch` after all.

Reply:
Yes, currently only x86 is working...